### DISCUSSION TITLE
Redundant 400-H get_cpu_state() returning incorrect value.

### DISCUSSION THREAD

**Discussion by sinamics** [gijzelaerr/python-snap7#473] [Q&A] [answered]
Hi
I'm dealing with a redundant 400-H system, and I'm using the [get_cpu_state()](https://python-snap7.readthedocs.io/en/latest/API/client.html#snap7.client.Client.get_cpu_state) function. 

However, I'm encountering an issue where the returned value is `S7CpuStatusStop`, even though both CPUs are in the RUN state. Interestingly, when only one CPU is in RUN, the returned value is correct, showing `S7CpuStatusRun`.

**Comment by gijzelaerr** [comment, marked as answer]
Closing as stale. No follow-up or reproduction details were provided for this niche redundant 400-H issue.
